main.py

```diff
@@ -1,10 +1,48 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+    for value in players.values():
+        if value["guild"] is not None:
+            if not Guild.objects.filter(name=value["guild"]["name"]).exists():
+                Guild.objects.create(
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places